src/test/groovy/com/codebrig/arthur/observe/structure/filter/RoleFilterTest.groovy

```diff
@@ -4,12 +4,21 @@ import com.codebrig.arthur.ArthurTest
 import com.codebrig.arthur.SourceLanguage
 import com.codebrig.arthur.observe.structure.filter.loop.ForLoopFilter
 import gopkg.in.bblfsh.sdk.v1.protocol.generated.Encoding
+import org.junit.After
+import org.junit.Before
 import org.junit.Test
 
 import static org.junit.Assert.assertTrue
 
 class RoleFilterTest extends ArthurTest {
 
+    ForLoopFilter forLoopFilter
```

Comment:
I don't think this needs to be a field. Looks like it would work as a function variable.